futures-core/src/stream/stream_obj.rs

```diff
@@ -0,0 +1,248 @@
+use crate::task::{self, Poll};
+use core::fmt;
+use core::marker::{PhantomData,Unpin};
+use core::mem::PinMut;
+
+use crate::stream::Stream;
```

Comment:
Move this after the first line. `use crate` first, then the others (and every line ordered alphabetically 🙂)